tests/phpunit/tests/test-strings.php

```diff
@@ -289,4 +289,18 @@ public function test_import_empty_string() {
 		$mo->import_from_db( $lang );
 		$this->assertSame( array( 'test', 'test 2' ), array_keys( $mo->entries ) );
 	}
+
+	public function test_translate_untranslated_string_should_return_original_string() {
+		$pll_admin = new PLL_Admin( $this->links_model );
+		$pll_admin->init();
+
+		$lang = self::$model->get_language( 'en' );
+		$mo   = new PLL_MO();
+		$mo->add_entry( $mo->make_entry( 'test', '' ) );
+		$mo->export_to_db( $lang );
+
+		do_action( 'pll_language_defined' );
```

Comment:
Why admin context? It's not useful to update strings in DB.
The action `pll_language_defined` without defining a language ?
If I look at previous tests, this is the kind of things usually done: https://github.com/polylang/polylang/blob/3d9bc5cd01f9dd5461e5783e62dd3ea86e0f9493/tests/phpunit/tests/test-strings.php#L198-L200

Reply:
Because the issue #2368 was revealed in a admin context (not frontend) with a ACF compatibility issue and also without a specific language defined (default WordPress locale one).

To be closer to the reality, I propose to trigger `setup_theme` action as Polylang `init_user()` is hooked on.
Thus `pll_no_language_defined` is done instead of `pll_language_defined`.
Then we also trigger `PLL_Base::load_strings_translations()` through `PLL_OLT_Manager::load_textdomains()` hooked on `pll_no_language_defined`.

We can also use `pll_no_language_defined` if you prefer.